docs/articles/user-guide/cli-commands.md

```diff
@@ -23,31 +23,39 @@ These Salesforce CLI commands are available in Salesforce Extensions for VS Code
 - `force:apex:test:run --resultformat human ...`: **SFDX: Invoke Apex Tests...**
 - `force:apex:trigger:create ...`: **SFDX: Create Apex Trigger**
 - `force:auth:logout --all --noprompt`: **SFDX: Log Out from All Authorized Orgs**
-- `force:auth:web:login --setdefaultdevhubusername`: **SFDX: Authorize a Dev Hub**
+- `force:auth:web:login --setdefaultdevhubusername`, `force:auth:device:login` (in remote development): **SFDX: Authorize a Dev Hub**
 - `force:config:list`: **SFDX: List All Config Variables**
+- `force:config:set --isvebuggerSid --isvDebuggerUrl --instanceUrl`: **Create and Set Up Project for ISV Debugging** (step 2)
```

Comment:
The `SFDX: Create and Set Up Project for ISV Debugging` executes several cli commands. Do we want to document every single command executed under this command label @ntotten ?